### PR TITLE
Improvements to the circleci config file

### PR DESCRIPTION
**Pull request by AlexandraMoga**
A new attempt at setting up CircleCi. Hope this works now. 

**Edit:** The build was successful. For the moment, I've only used one test in the run command to make sure that the environment works. I will add more tests after review. 

**Update**: 
- renamed `build` job as `test` in config.yml
- updated the install tests and added some extra wait to avoid intermittent test failures
- added more tests to run on CI: tests currently running are search, home and install

### CODE REVIEW

**Comment by jrbenny35** [comment]
I think since the build is successful we should just change this PR to allow for all the tests to run.

File: .circleci/config.yml [hunk 1]
```diff
@@ -5,7 +5,8 @@ jobs:
   build:
```
Comment (jrbenny35) [inline .circleci/config.yml:5]: Can we name this ```test``` instead? `

**Review by jrbenny35 — CHANGES_REQUESTED** [review]
Awesome work!

File: .circleci/config.yml (hunk 1 above)
Comment (AlexandraMoga) [reply .circleci/config.yml]: renamed as `test`

File: test_login.py [hunk 2]
```diff
@@ -1,9 +1,8 @@
 import pytest
-
 from pages.desktop.home import Home
 
 
-
+@pytest.mark.skip
```
Comment (jrbenny35) [inline test_login.py:5]: Can we add a reason here, just so we remember why we skipped. https://docs.pytest.org/en/latest/skipping.html#skipping-test-functions

File: test_login.py [hunk 3]
```diff
@@ -13,6 +12,7 @@ def test_login(base_url, selenium, fxa_account):
     assert page.logged_in
 
 
+@pytest.mark.skip
```
Comment (jrbenny35) [inline test_login.py:15]: Can we add a reason here, just so we remember why we skipped. https://docs.pytest.org/en/latest/skipping.html#skipping-test-functions

**Review by jrbenny35 — APPROVED** [review]
r+, wc

Looks great! Added some final requests but once you address them you can merge.

**Pull request merged**